checkov/secrets/scan_git_history.py

```diff
@@ -153,13 +157,15 @@ def _run_scan_one_commit(commit: Dict[str, str | Dict[str, str]]) -> Tuple[List[
                     f"Found {len(file_results)} secrets in file path {file_name} in commit {commit_hash}")
                 results.append(RawStore(file_results=file_results, file_name=file_name, commit=commit,
                                         commit_hash=commit_hash, type=FILE_RESULTS_STR,
-                                        rename_from='', rename_to=''))
+                                        rename_from='', rename_to='', committer=commit.get(COMMIT_COMMITTER),
+                                        committed_datetime=commit.get(COMMIT_COMMITTED_DATETIME)))
```

Comment:
maybe we should put all metadata under one key ? 
meta: date create, date delete, commiter, hash ?

Reply:
I can add another `TypedDict`
I thought `EnrichedPotentialSecretMetadata` can be enough
change it here too?

Reply:
I mean: currently the commit wiill look like 
{ hash: 'fsafsags', date: '2010..',  added_by: 'maryna' ... 'file.py': 'some diffs', 'file2.py': 'some diffs' }
what I mean is to do:
{ commit_meta: { hash: 'fsafsags', date: '2010..',  added_by: 'maryna' ...} 
files: 'file.py': 'some diffs', 'file2.py': 'some diffs' }

don't have to add the 'files' key, but the others can be united under one key and then it's more consolidated